techniques/system/common/1.0/environment-variables.st

```diff
@@ -27,7 +27,8 @@ bundle agent get_environment_variables
     # will be available using the syntax ${node.env[ENV_NAME]}
 
     !windows::
-      "script" string => "#! /bin/sh
+      "script" string => "#!/bin/sh
+source /etc/profile
```

Comment:
This should be "." not source, because "source" is a bashism.

Also, maybe we should check that /etc/profile actually exists first, like ```[ -f /etc/profile ] && . /etc/profile```